Dumpers/python/Cfg_RecoSimDumper_cfi.py

```diff
@@ -38,6 +38,7 @@
     saveSimhits                       = cms.bool(True),  #save simHits information
     saveRechits                       = cms.bool(True),  #save recHits information
     savePFRechits                     = cms.bool(True),  #save pfRecHits information
+    saveEBPFRechits                   = cms.bool(False),  
```

Comment:
@mgratti 

Out of curiosity, can you confirm if my understanding is correct:

- savePFRechits: saves the pfRecHits information for both EE and EB
- saveEBPRRecHits: saves the pfRecHits information only for EB (which is only what you needed, as you run the GNN study on barrel only?)
- --> if both options are set to true, there would be duplicated information for the barrel (?), therefore it is recommended to set only one to true at a time ?

Reply:
in reality the option savePFRechits saves unclustered PFRechit if I remember correctly, while saveEBPFRechits saves all EB PFrechits